app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<section class="reg-container">
+  <h2>Log in</h2>
+
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name), html: {class: 'form-control'}) do |f| %>
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password %><br />
+      <%= f.password_field :password, autocomplete: "current-password" %>
+    </div>
+
+    <% if devise_mapping.rememberable? %>
+      <div class="field">
+        <%= f.check_box :remember_me %>
+        <%= f.label :remember_me %>
+      </div>
+    <% end %>
+
+    <div class="actions">
+      <%= f.submit "Log in", class:'btn btn-block'%>
+    </div>
+  <% end %>
+
```

Comment:
- [ ] Well done for implementing the styles ✨👏 However, as suggested by the reviewer, kindly make sure to use `placeholders` instead of `labels` for input fields as given in the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_login.png?token=GHSAT0AAAAAAB5V373UNEX72ZQIIXRCIWB2ZGCO73A)